app/models/work.rb

```diff
@@ -1,35 +1,26 @@
 class Work < ApplicationRecord
   include Taggable
-  include Creatable
+  include CreationNotifier
   include Collectible
   include Bookmarkable
-  include Pseudable
   include Searchable
   include BookmarkCountCaching
   include WorkStats
   include WorkChapterCountCaching
   include ActiveModel::ForbiddenAttributesProtection
+  include Creatable
 
   ########################################################################
   # ASSOCIATIONS
   ########################################################################
 
-  # creatorships can't have dependent => destroy because we email the
-  # user in a before_destroy callback
-  has_many :creatorships, as: :creation
-  has_many :pseuds,
-           through: :creatorships,
-           after_remove: [:expire_pseud, :reindex_changed_pseud]
```

Comment:
The method `reindex_changed_pseud` should be removed.

Reply:
Good catch! I also removed `expire_pseud`, since it's in the same boat.